src/components/SQForm/SQFormAutocomplete.js

```diff
@@ -144,7 +144,7 @@ function SQFormAutocomplete({
   const handleInputChange = React.useCallback(
     (_event, value) => {
       setInputValue(value);
-      onInputChange && onInputChange();
+      onInputChange && onInputChange(_event, value);
```

Comment:
I don't think we need the `_event` to be passed.  It looks like the value is all a consumer would need.  
![image](https://user-images.githubusercontent.com/20329437/104622551-091d7100-5657-11eb-9552-d14b1b99c007.png)

Reply:
> I don't think we need the `_event` to be passed. It looks like the value is all a consumer would need.
> ![image](https://user-images.githubusercontent.com/20329437/104622551-091d7100-5657-11eb-9552-d14b1b99c007.png)

Generally, the value is what we need.
However, for any generic event handlers, the expected API is for the first argument to be the `event`

The 2nd argument `value` is  a nicety from `formik` since the most common use from an event handler is to use `event.target.value` to Chris's point.

Our component `onChange` and `onClick` event handlers should include the `event` as the first argument.

In the case we should include the value as the second argument to maintain the same API